File: Week_01/Mario.c
/**********************************************************************/
/*                          Week 1 : Mario-More                       */
/*                                                                    */
/*   Implement a program in C that prompts the user for their name    */
/*                 and then says hello to that user.                  */
/**********************************************************************/
#include <cs50.h>
#include <stdio.h>

int main() {
    int n;
    do{
        n = get_int("Heights : ");
    }while((n <= 0) || (n > 9));
    for(int i = 1; i <= n; i++){
        for(int j = 0;j < (n-i); j++){
            printf(" ");
        }
        for(int k = 0;k < i; k++){
            printf("#");
        }
        printf("  ");
        for(int k = 0;k < i; k++){
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: Week_06/Mario.py
########################################################################
#                       Week 6 : Mario                                 #
#                                                                      #
#    Implement a program in C that prompts the user for their name     #
#                and then says hello to that user.                     #
########################################################################

from cs50 import get_int

height = 0
while height <= 0 or height >= 9:
    height = get_int("Height: ")

for i in range(height):
    print(" " * (height-i-1), end="")

    print("#" * (i+1), end="")

    print(" "*2, end="")

    print("#" * (i+1), end="")

    print("")


